### PR TITLE
docs: adds docs tool, material for mkdocs, along with more fleshed ou…

### PR DESCRIPTION
**Pull request by jaydrennan**
…t docstrings.

this includes ability to serve up a local docs website.

### CODE REVIEW

**Review by brycedrennan — CHANGES_REQUESTED** [review]
Delete most the stuff chatgpt made. It highlights unimportant details.

File: docs/assets/000078_260972468_PLMS40_PS7.5_portrait_photo_of_a_freckled_woman.jpg
Comment (brycedrennan) [inline docs/assets/000078_260972468_PLMS40_PS7.5_portrait_photo_of_a_freckled_woman.jpg:1]: Lets make a new more interesting image and use it here.

**Comment by brycedrennan** [comment]
`make docs`
```
mkdocs serve
make: mkdocs: No such file or directory
make: *** [docs] Error 1
```

File: docs/assets/000078_260972468_PLMS40_PS7.5_portrait_photo_of_a_freckled_woman.jpg
Comment (jaydrennan) [reply docs/assets/000078_260972468_PLMS40_PS7.5_portrait_photo_of_a_freckled_woman.jpg:1]: Replaced it with something different, but not necessarily better.

**Comment by jaydrennan** [comment]
> Delete most the stuff chatgpt made. It highlights unimportant details.

I believe this has been completed.

**Comment by jaydrennan** [comment]
> `make docs`
> 
> ```
> mkdocs serve
> make: mkdocs: No such file or directory
> make: *** [docs] Error 1
> ```

This is probably because I didn't add necessary pip requirements.

**Comment by brycedrennan** [comment]
```
mkdocs serve
ERROR   -  Config value 'plugins': The "mkdocstrings" plugin is not installed

Aborted with 1 configuration errors!
make: *** [docs] Error 1
```

**Comment by jaydrennan** [comment]
> ```
> mkdocs serve
> ERROR   -  Config value 'plugins': The "mkdocstrings" plugin is not installed
> 
> Aborted with 1 configuration errors!
> make: *** [docs] Error 1
> ```

Missed a package. They're all there now.

**Review by brycedrennan — APPROVED** [review]

**Pull request merged**